src/pretix/control/templates/pretixcontrol/event/settings.html

```diff
@@ -99,10 +103,80 @@ <h1>{% trans "General settings" %}</h1>
                 {% bootstrap_field sform.frontpage_text layout="control" %}
                 {% bootstrap_field sform.presale_has_ended_text layout="control" %}
                 {% bootstrap_field sform.voucher_explanation_text layout="control" %}
-                {% bootstrap_field sform.confirm_text layout="control" %}
                 {% bootstrap_field sform.checkout_email_helptext layout="control" %}
                 {% bootstrap_field sform.banner_text layout="control" %}
                 {% bootstrap_field sform.banner_text_bottom layout="control" %}
+            
+                <div class="form-group">
```

Comment:
Can we move this back up, preferrably even just below ``frontpage_text``, since it's arguably the most important thing on this page.

Reply:
Yes we can.